### PR TITLE
Added my first solution.

### PR DESCRIPTION
**Pull request by vadimkik2**

### CODE REVIEW

**Review by Maksym-Korkosh — APPROVED** [review]

**Review by Mihael0204 — APPROVED** [review]

File: src/main/java/taxi/controller/authentication/LogoutController.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LogoutController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        request.getSession().invalidate();
+        response.sendRedirect("/login");
```
Comment (Sofasmile) [inline src/main/java/taxi/controller/authentication/LogoutController.java:14]: use context path

**Review by Sofasmile — CHANGES_REQUESTED** [review]
fix: 
- drivers/add.jsp
- add changes to web.xml

**Review by Sofasmile — CHANGES_REQUESTED** [review]
image

File: src/main/webapp/WEB-INF/views/drivers/add.jsp [hunk 2]
```diff
@@ -23,6 +26,12 @@
         <td>
             <input type="text" name="license_number" form="driver" required>
         </td>
+        <td>
+            <input type="text" name="login" form="driver" required>
+        </td>
+        <td>
+            <input type="text" name="password" form="driver" required>
```
Comment (Sofasmile) [inline src/main/webapp/WEB-INF/views/drivers/add.jsp:33]:
```suggestion
            <input type="password" name="password" form="driver" required>
```

**Review by Sofasmile — CHANGES_REQUESTED** [review]

**Pull request closed without merging**